feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.spec.ts

```diff
@@ -182,5 +182,37 @@ describe('ConfigUIKeyGeneratorService', () => {
       const techLabel = label + ' / [' + techName + ']';
       expect(classUnderTest.getLabel(true, label, techName)).toEqual(techLabel);
     });
```

Comment:
Plz, add a test for zero price. It expects `label` as return value and not `label [+$0.0]`.

Reply:
done